nusacrowd/nusa_datasets/indotacos/indotacos.py

```diff
@@ -0,0 +1,137 @@
+from pathlib import Path
+from typing import List
+
+import datasets
+
+from nusacrowd.utils.configs import NusantaraConfig
+from nusacrowd.utils.constants import Tasks
+from nusacrowd.utils import schemas
+
+import pandas as pd
+
+_CITATION = """\
+    author = "Christian Wibisono"
```

Comment:
Maybe we can change it to this one:

``` 
@misc{wibisono2022indotacos,
  title = {IndoTacos},
  howpublished = {\\url{https://www.kaggle.com/datasets/christianwbsn/indonesia-tax-court-verdict}},
  note = {Accessed: 2022-09-22}
}
```

Reply:
I have changed the citation accordingly, please kindly check. Thank you!